src/unleash.ts

```diff
@@ -50,6 +53,12 @@ export interface StaticContext {
 }
 
 export class Unleash extends EventEmitter {
+  private static configSignature?: string;
+
+  private static instance?: Unleash;
+
+  private static instanceCount: 0;
```

Comment:
Oh, this is a very nice bug, I accidentally set the type to "0" which is definetley not what I tried to do here!